node_modules/gh-series/lib/internal/patterns/cambridge/daytime.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (c) 2014 "Fronteer LTD"
+ * Grasshopper Event Engine
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+var _ = require('lodash');
+var util = require('util');
+
+var moment = require('moment-timezone');
+
+/**
+ * The model that holds information about when an event takes place during the day
+ *
+ * @param  {Moment}     start       The start time of the event
+ * @param  {Moment}     end         The end time of the event
+ */
+var DayTime = module.exports.DayTime = function(start, end) {
+
+    var that = {
+        // Which day of the week the event starts, Sunday = 0
+        'dayOfWeek': start.weekday(),
+
+        // The hour the event takes place *in Cambridge (UK)*. As this will eventually get
+        // stringifyfied, we use the time in Cambridge (UK)
+        'startHour': moment.tz(start, 'Europe/London').hour(),
+        'startMinute': start.minute(),
+        'endHour': moment.tz(end, 'Europe/London').hour(),
+        'endMinute': end.minute()
+    };
+
+    /**
+     * Compare this daytime instance to another
+     *
+     * @param  {DayTime}    other       The daytime instance to compare to
+     * @return {Boolean}                `true` if the other daytime instance is the same
```

Comment:
It doesn't have to be the same instance, it just needs to be an event at the same time